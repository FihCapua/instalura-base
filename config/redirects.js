module.exports = [
  {
    source: '/404/',
    destination: '/error/404',
    permanent: true,
  },
  {
    source: '/login/',
    destination: '/app/login/',
    permanent: true,
  },
];
