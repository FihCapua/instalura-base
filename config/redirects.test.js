import redirects from './redirects';

console.log(redirects);
describe('config/redirects', () => {
  test('renders all current redirects', () => {
    expect(redirects).toMatchSnapshot();
  });
});
