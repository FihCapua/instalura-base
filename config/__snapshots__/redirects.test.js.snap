// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/redirects renders all current redirects 1`] = `
Array [
  Object {
    "destination": "/error/404",
    "permanent": true,
    "source": "/404/",
  },
  Object {
    "destination": "/app/login/",
    "permanent": true,
    "source": "/login/",
  },
]
`;
